examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,368 @@
+# Kubernetes Deployment Guide
+
+## Setup Environment
+
+1. Start a Kubernetes cluster with version 1.15 or higher and at least 4GB of memory
+
+```
+$ minikube start --kubernetes-version v1.15.4
+😄  minikube v1.4.0 on Darwin 10.14.6
+🔥  Creating virtualbox VM (CPUs=2, Memory=4096MB, Disk=20000MB) ...
+🐳  Preparing Kubernetes v1.15.4 on Docker 18.09.9 ...
+🚜  Pulling images ...
+🚀  Launching Kubernetes ...
+⌛  Waiting for: apiserver proxy etcd scheduler controller dns
+🏄  Done! kubectl is now configured to use "minikube"
+```
+
+2. Navigate into the directory where all the kuma-demo YAML files are:
+
+```
+$ cd examples/kubernetes/kuma-demo/
+```
+
+3. Deploy Kuma's sample marketplace application
+
+```
+$ kubectl apply -f kuma-demo-aio.yaml
+namespace/kuma-demo created
+serviceaccount/elasticsearch created
+service/elasticsearch created
+replicationcontroller/es created
+deployment.apps/redis-master created
+service/redis-master created
+service/kuma-demo-api created
+deployment.apps/kuma-demo-app created
+```
+
+This will deploy our demo marketplace application split across 3 pods. The first pod is an Elasticsearch service that stores all the items in our marketplace. The second pod is our Node/Vue application that allows you to visually query the Elastic and Redis endpoints. The last pod is a Redis service that stores reviews for each item.
+
+Check the pods are up and running by checking the `kuma-demo` namespace
+
+```
+$ kubectl get pods -n kuma-demo
+NAME                            READY   STATUS    RESTARTS   AGE
+es-n8df7                        1/1     Running   0          13m
+kuma-demo-app-8fc49ddbf-gfjtb   2/2     Running   0          13m
+redis-master-6d4cf995c5-nsghm   1/1     Running   0          13m
+```
+
+In the following steps, we will be using the pod name of the `kuma-demo-app-*************` pod. Please replace any `${KUMA_DEMO_APP_POD_NAME}` variables with your pod name.
+
+4. Deploy the logstash service.
```

Comment:
I would suggest to move this step right before applying `Mesh` policy that points to that logging backend.

Reply:
done, thanks for the review